# Files used for determining the 18S sequencing bias for GA03 and GP13 cruises

In order to determine the Illumina sequencing bias associated with longer 18S reads, we compared Bioanalyzer plots from the pooled samples (reflecting the ratio of 16S:18S in PCR products) with the results from the sequencer (using raw reads that were not denoised, only split into 16S and 18S pools; see [here](https://github.com/jcmcnch/eASV-pipeline-for-515Y-926R) for more info). Using this information, we determined a mean 5.76x sequencing bias against longer 18S reads (see `201106_bias_calculation.ods` for details). This bias is consistent with the slope of the line in the plot `201112_MGEukFrac_vs_tag_EukFrac_BGT_GA03-GP13.png`, which compared the 18S fraction between amplicons and metagenomes, suggesting our estimated bias is roughly accurate. Metagenomic 18S fractions were determined with [phyloFlash](https://github.com/HRGV/phyloFlash).
